src/activities/space-encounter/SpaceEncounter.qml

```diff
@@ -0,0 +1,178 @@
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.1
+
+import "../../core"
+import "space-encounter.js" as Activity
+
+ActivityBase {
+    id: activity
+
+    onStart: focus = true
+    onStop: {}
+
+    pageComponent: Image {
+        id: background
+        //width: parent; height: parent;
+        //////////////////////////////////////////////////////////////////////////////
+        fillMode: Image.PreserveAspectCrop
+        source: "./resource/asteroid.svg"
+        sourceSize.width: parent.width
+        //anchors.fill: parent
+        //color: "#ABCDEF"
+        /////////////////////////////////////////////////////////////////////////////////
+        signal start
+        signal stop
+
+        Component.onCompleted: {
+            activity.start.connect(start)
+            activity.stop.connect(stop)
+        }
+
+        // Add here the QML items you need to access in javascript
+        QtObject {
+            id: items
+            property Item main: activity.main
+            property alias background: background
+            property alias bar: bar
+            property alias bonus: bonus
+            /////////////////////////////////////////////////////////////////////////
+            /* Variables */
+            property bool flag: true
+            property bool timerRunning: false
+            property bool timerRepeat: true
+
+            property int spacePresses: 0
+            property int ticks: 0
+            property int highScore: 0
+            property int playerScore: 0
+
+            property string textbuttText: "Press Space Bar to Start"
+            property string displayCounterText: items.ticks + " Seconds"
+            property string spaceBarButtonColor: "#ABCDEF"
+            //////////////////////////////////////////////////////////////////////////////////////////////
+        }
+
+        onStart: { 
+            Activity.start(items)
+            textbutt.forceActiveFocus()
+        }
+        onStop: { Activity.stop() }
+
+        
+/////////////////////////////////////////////////////////////////
+        Rectangle {
+            id: spaceBarButton
+            anchors.centerIn: parent;
+            width: 750; height: 100;
+            color: items.spaceBarButtonColor;
+            GCText {
+                id: textbutt
+                anchors.centerIn: parent
+                fontSize: largeSize
+                text: items.textbuttText;
+
+                focus: true
+                Keys.onPressed: {
+                    if((event.key == Qt.Key_Space)&&(event.isAutoRepeat == 0)) {
```

Comment:
event.isAutoRepeat is a boolean, you can use true/false or just !event.isAutoRepeat